### PR TITLE
Add pulpcore 3.49 pipelines

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

File: theforeman.org/pipelines/vars/pulpcore/3.49.groovy [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def pulpcore_version = '3.39'
+def pulpcore_distros = ['el8', 'el9']
+def packaging_branch = 'rpm/3.39'
```
Comment (evgeni) [inline theforeman.org/pipelines/vars/pulpcore/3.49.groovy:3]: 1234!

**Review by evgeni — CHANGES_REQUESTED** [review]
see inline

**Comment by Odilhao** [comment]
Nothing to see here

**Review by evgeni — APPROVED** [review]

**Comment by Odilhao** [comment]
As always, I don't have merge permissions, and thats amazing

**Pull request merged**